pkg/cmd/server/origin/master.go

```diff
@@ -582,6 +573,23 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		"clusterRoles":          clusterRoleStorage,
 	}
 
+	if sdnovs.IsOpenShiftNetworkPlugin(c.Options.NetworkConfig.NetworkPluginName) {
```

Comment:
When user switches from openshift network plugin to third party plugin, oc get clusternetwork/hostsubnets provides stale info and it will be confusing(clusternetwork will show multitenant/subnet pluginName when they are using a different one). I didn't prefer to delete  existing clusternetwork/hostsubnets because any unintentional typo in plugin name will wipe out the configuration. 
This is similar to storage for builds, build endpoints are enabled based on user config.These REST endpoints are specific to our SDN and if users are not using our plugin they shouldn't care about these endpoints? 
@smarterclayton